demisto_sdk/commands/content_graph/parsers/related_files.py

```diff
@@ -278,16 +278,29 @@ class ImageRelatedFile(PNGFiles):
     file_type = RelatedFileType.IMAGE
 
     def get_optional_paths(self) -> List[Path]:
-        return [
+        """
+        Generate a list of optional paths for the content item's image.
+        The right path will be found later.
+        Returns:
+            List[Path]: the list of optional paths.
+        """
+        optional_paths_list = [
             self.main_file_path.parents[1]
             / "doc_files"
             / str(self.main_file_path.parts[-1])
             .replace(".yml", ".png")
-            .replace("playbook-", ""),
-            Path(str(self.main_file_path).replace(".yml", ".png")),
-            self.main_file_path.parent / f"{self.main_file_path.parts[-2]}_image.png",
-            Path(str(self.main_file_path).replace(".json", "_image.png")),
+            .replace("playbook-", ""),  # In case the playbook's image is located under doc_files with the same name as the playbook.
```

Comment:
```suggestion
            .replace("playbook-", ""),  # In case the playbook's image is located under doc_files folder with the same name as the playbook.
```